static_files/css/style.css

```diff
@@ -7,7 +7,34 @@ html {
 }
 
 @import url('https://fonts.googleapis.com/css2?family=Dongle:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Chenla&family=Dongle:wght@300&display=swap');
 
 body{
     font-family: "Dongle", sans-serif;
+    min-height: 100vh;
 }
+
+header{
+    min-height: 50px;
+}
+
+body{
+    display: flex;
+    flex-direction: column;
+}
+
+main{
+    flex: 1;
+}
+
+.Footer {
+    min-height: 50px;
+    font-family: "Nunito", sans-serif;;
```

Comment:
You use the nunito source without any import in the local css files or from an external css through a <link/> tag included in the <head> of the page and this is a problem. I suggest that you include an import of the Nunito source. Already thinking about implementing the requested adjustment, I launch a challenge for you: With or without an import of the Nunito font, the font style will possibly remain the same, why does this happen? (Spoiler (since the tough question lol): it involves the browser)